main.go

```diff
@@ -128,7 +129,7 @@ func main() {
 		logger.Info().Msgf("error getting ethereum address: %s", err)
 		_ = ds.SendErrorPayload(errors.Wrap(ethErr, "could not get device eth addr"), nil)
 	}
-	vehicleSignalDecodingAPI := gateways.NewVehicleSignalDecodingAPIService()
+	vehicleSignalDecodingAPI := gateways.NewVehicleSignalDecodingAPIService(ENV)
```

Comment:
how about further up here we also log what env was used? I could see us using it later on also to set log level from DEBUG (dev) to INFO (prod)